src/Worker/LASWorker.js

```diff
@@ -0,0 +1,14 @@
+import LASLoader from 'Parser/LASLoader';
+import { expose } from 'threads/worker';
+
+const loader = new LASLoader();
+
+expose({
+    set lazPerf(path) {
```

Comment:
Why do you need to exposer `set lazPerf`?

Reply:
Wanted to expose it to set the `laz-perf.wasm` in the same way that with the non-worker loader!